### PR TITLE
fix(prowlarr): fix typo in prowlarr-setup-proxy.md

### PR DESCRIPTION
**Pull request by rangzen**
# Pull Request

## Purpose

Fix some typos in Prowlarr, how to setup proxy.

## Requirements

- [x] These changes meet the standards for [contributing](https://github.com/TRaSH-Guides/Guides/blob/master/CONTRIBUTING.md).
- [x] I have read the [code of conduct](https://github.com/TRaSH-Guides/Guides/blob/master/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

**Comment by rangzen** [comment]
Do I have to add myself in Contributors?

**Comment by nuxencs** [comment]
no that's a bug in the workflow

File: docs/Prowlarr/prowlarr-setup-proxy.md [hunk 1]
```diff
@@ -12,13 +12,13 @@ Just run VPN on the apps that actually need it, like your torrent client, and pe
 
     This isn't the same as the tracker, the API/Website scraping happens on the actual site and some indexers don't allow VPN usage on their site.
 
-In this guide i will try to explain how to setup Prowlarr to use your VPN from your torrent client.
+In this guide, I will try to explain how to setup Prowlarr to use your VPN from your torrent client.
```
Comment (zakkarry) [inline docs/Prowlarr/prowlarr-setup-proxy.md:15]: I'm not sure, but I seem to recall we switched from an "I" to "We" in the guides going forward. If not my bad.

File: docs/Prowlarr/prowlarr-setup-proxy.md [hunk 2]
```diff
@@ -12,13 +12,13 @@ Just run VPN on the apps that actually need it, like your torrent client, and pe
...
 
 ## Enable privoxy
 
-If you use docker you need to enable enable privoxy in you torrent client so you can use it to passthrough traffic.
+If you use docker you need to enable privoxy in your torrent client so you can use it to passthrough traffic.
 
-Please check your used docker image how to enable privoxy.
+Please check your used docker image on how to enable privoxy.
```
Comment (zakkarry) [inline docs/Prowlarr/prowlarr-setup-proxy.md:21]:
```suggestion
Please check your docker image's documentation on how to enable privoxy.

File: docs/Prowlarr/prowlarr-setup-proxy.md [hunk 3]
```diff
@@ -12,13 +12,13 @@ Just run VPN on the apps that actually need it, like your torrent client, and pe
 
     This isn't the same as the tracker, the API/Website scraping happens on the actual site and some indexers don't allow VPN usage on their site.
 
-In this guide i will try to explain how to setup Prowlarr to use your VPN from your torrent client.
+In this guide, I will try to explain how to setup Prowlarr to use your VPN from your torrent client.
 
 ## Enable privoxy
 
-If you use docker you need to enable enable privoxy in you torrent client so you can use it to passthrough traffic.
+If you use docker you need to enable privoxy in your torrent client so you can use it to passthrough traffic.
```
Comment (zakkarry) [inline docs/Prowlarr/prowlarr-setup-proxy.md:19]:
```suggestion
If you use docker, you need to enable privoxy in your torrent client so you can use it to pass through traffic.

File: docs/Prowlarr/prowlarr-setup-proxy.md [hunk 4]
```diff
@@ -49,7 +49,7 @@ Select the indexer that you want to use with Privoxy
 
 ![Select Indexer](images/select-indexer.png)
 
-Scroll down to the bottom and add the tag you've set up earlier at step 2.
+Scroll down to the bottom and add the tag you set up earlier at step 2.
```
Comment (zakkarry) [inline docs/Prowlarr/prowlarr-setup-proxy.md:52]:
```suggestion
Scroll down to the bottom and add the tag you set up earlier during step 2.

**Review by zakkarry — CHANGES_REQUESTED** [review]

File: docs/Prowlarr/prowlarr-setup-proxy.md (hunk 1 above)
Comment (TRaSH-) [reply docs/Prowlarr/prowlarr-setup-proxy.md:15]: We

Comment (zakkarry) [reply docs/Prowlarr/prowlarr-setup-proxy.md:15]: @nuxencs I changed it to a suggestion you can commit

**Pull request merged**